pages/linux/rmmod.md

```diff
@@ -5,20 +5,20 @@
 
 - Remove a module from the kernel:
 
-`sudo rmmob {{module_name}}`
+`sudo rmmod {{module_name}}`
 
 - Remove a module from the kernel and display verbose information:
 
-`sudo rmmob --verbose {{module_name}}`
+`sudo rmmod --verbose {{module_name}}`
 
 - Remove a module from the kernel and send errors to syslog instead of standard error:
 
-`sudo rmmob --syslog {{module_name}}`
+`sudo rmmod --syslog {{module_name}}`
 
 - Display help:
```

Comment:
```suggestion
- Print the help:
```

Reply:
We use display help in a lot of pages.

Reply:
I don't know whether we have to use `Display help` instead of my suggestion but I think we have to note about it in style guide to make all pages consistent. It's like [Approved Verbs for PowerShell Commands](https://docs.microsoft.com/en-us/powershell/scripting/developer/cmdlet/approved-verbs-for-windows-powershell-commands?view=powershell-7.2). [This](https://github.com/tldr-pages/tldr/pull/7441) PR tries to mention --help/--version examples. 😄